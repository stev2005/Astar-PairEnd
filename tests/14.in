CACGTCAACCGCATTAACCGGTTGAACCTATCACTTGTTAACCCCACC
15
GATCTCGTGTTGGATTTGGTGGTTT
ATGGAAGTGCAA
GTACCAACTGTGACGAAGATAGTCAGT
GTTAGT
TTCAGATCAATAACCTGTGTAAGGGCCTTAAGT
TCGGGGTCTTTATCTCATGGCATGTGGA
CGTTAGCGCTCAGCTTCCTATCGTAATGTTGACG
C
CTCACCGGCCTGAACCGACGTTCGCGTTCATGAAGC
ATTC
CCTGGCGTTGGCAATATAATTATCCCGCATA
CTTTAGTTACAACTACACA
CTATCGACGATTTGGTACGATTCAGCCGGCCACGT
ATTG
GCGCACCCTGAGTCC
