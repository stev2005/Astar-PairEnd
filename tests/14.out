12
6
11
2
15
13
15
0
14
1
14
7
16
1
7
